doc/nrf/ug_multi_image.rst

```diff
@@ -165,6 +165,11 @@ For example, to change the ``CONF_FILE`` variable for the MCUboot image and the
 
    cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
 
+The command line that is used to create the child images can be extended by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
+This could for instance be used to get more debug information with the flag ``---trace-expand``.
+
+Similarly the CMake variable ``EXTRA_MULTI_IMAGE_BUILD_OPT`` can be used to modify the options used when ninja is invoked on the child images.
```

Comment:
I would prefer to not use `ninja` here, but rather, something in line with: `the options used when the build command is invoked on the child images`.
as the user could be using `make` instead of ninja.

Although, I personally would prefer to inform the user of the `ninja -C <child_image_dir>` / `make -C <child_image_dir>` possibility, and then only introduce this extra argument if actually requested by users.

Reply:
"ninja" is less ambiguous than "the build command" and there is precedence for directly using ninja in the docs.

Reply:
https://docs.zephyrproject.org/latest/application/index.html

Reply:
but that page describes both `ninja` or `make`.
And states that `make` is an option on Linux and macOS, but for simplicity they will use `ninja` for rest of document.

Now, in this user guide we only use `ninja` and therefor I feel we should at least mention `make` as well.

@ru-fu Do you have an insight in the NCS doc on this. Do we in general only use `ninja` or do we also describe `make` as a possibility. 
To me, consistency is most important :)

Reply:
In the NCS docs, we don't use ninja at all (except for the documentation build). We mention CMake in a few places, but in general we focus on SES. We are currently adding information about command line builds, but there we'll focus on west.
So I'm not in favor of mentioning ninja here.
That being said, I didn't get around to reviewing the addition earlier, so I have some more suggestions anyway. ;)

Reply:
Is it possible to do the `ninja -C <sub_image> -v` option for `SES`? If not this option may still be needed.

Reply:
For SES right now, we might need an update.
Cause for parent image, SES is not invoking `ninja`, instead they parse the ninja files generated and invoke the compiler directly to have the output directly into SES.
For child images i'm unsure what will be done in long term (later updates to SES), but currently the build of sub-images are handled through CMake / ninja cause they are external projects, so there is kind of a mix of principles there, and SES know nothing about the sub-image.

So for SES, this could be a way to get verbose output, although user would have to go through
`Tools -> Options -> nRF Connect -> Additional CMake Options` and then re-import the project.
And then the same when disabling verbose output again.

Not even a very friendly solution in SES either.

Reply:
@tejlmand : What if I change from "ninja" to "ninja or make" ?

I don't feel that this is necessary, but this appears to be subjective.

Reply:
I added a suggestion further down that doesn't mention ninja or cmake. Maybe that'll do?

Reply:
I don't think people know what a build command is, but since both you and @tejlmand support this phrasing I'll trust your judgement.

Reply:
Will create another PR to address doc issues.

Reply:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1645